receiver/googlecloudspannerreceiver/config.go

```diff
@@ -29,10 +29,11 @@ const (
 type Config struct {
 	scraperhelper.ScraperControllerSettings `mapstructure:",squash"`
 
-	TopMetricsQueryMaxRows int       `mapstructure:"top_metrics_query_max_rows"`
-	BackfillEnabled        bool      `mapstructure:"backfill_enabled"`
-	CardinalityTotalLimit  int       `mapstructure:"cardinality_total_limit"`
-	Projects               []Project `mapstructure:"projects"`
+	TopMetricsQueryMaxRows      int       `mapstructure:"top_metrics_query_max_rows"`
+	BackfillEnabled             bool      `mapstructure:"backfill_enabled"`
+	CardinalityTotalLimit       int       `mapstructure:"cardinality_total_limit"`
+	Projects                    []Project `mapstructure:"projects"`
+	HideTopnQuerystatsQuerytext bool      `mapstructure:"hide_topn_querystats_querytext"`
```

Comment:
This is the only metric with query text.
The reason why I named it HideTopnQueryStatsQueryText is because of the categorization of metrics, arising from https://cloud.google.com/spanner/docs/introspection. TLDR : There are 2 types of tables : TopN and Total --- for each of QueryStats/ReadStats/TransactionStats/LockStats. QueryText is one of the fields of the TopN Query Stats table -- hence the name Hide-Topn-QueryStats-QueryText

Reply:
Makes sense.  Is there a chance you will ever want to add other metrics/tables with query_text in the future?  If not, the name being very specific is probably OK